int32[] class_ids

float32[] score

int32[] x1
int32[] y1
int32[] x2
int32[] y2
